src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -13437,8 +13881,8 @@ protected internal override void EnumerateTemplates(Func<string, string, EventFi
 
                 // New style
                 templates[22] = TieredCompilationSettingsDCStartTemplate(null);
-
                 templates[23] = ExecutionCheckpointDCEndTemplate(null);
+                templates[24] = GCSettingsRundownTemplate(null);
```

Comment:
just a general question - so the new style doesn't require the event id and all that? can all event data be specified this way? @brianrob

Reply:
All of the information still has to be specified - it's just specified in a different spot.  For this event it's specified here: https://github.com/microsoft/perfview/pull/1758/files#diff-5a4cce33f428c644a7af2b6ce4e0f6eff4f05f9e81ace26b0f145357bd61775fR13850

Reply:
that just takes me to Andrew's commit 😀 which lines in which file should I be looking at?

Reply:
Weird.  The link does actually point to the line in question, but it didn't expand `ClrTraceEventParser.cs`.  Check out line13850.